cpp/src_prims/timeSeries/jones_transform.h

```diff
@@ -0,0 +1,483 @@
+
+
+/*
+ * Copyright (c) 2019, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/**
+* @file jones_transform.h
+* @brief Transforms params to induce stationarity/invertability.
+* reference: Jones(1980) 
+*/
+
+#include <math.h>
+#include <cub/cub.cuh>
+#include "common/cuml_allocator.hpp"
+#include "common/device_buffer.hpp"
+#include "cuda_utils.h"
+#include "linalg/unary_op.h"
+
+
+namespace MLCommon {
+
+namespace TimeSeries {
+
+/**
+* @brief Lambda to map to the partial autocorrelation
+*
+* @tparam Type: Data type of the input 
+* @param in: the input to the functional mapping
+* @return : the Partial autocorrelation (ie, tanh(in/2))
+*/
+template <typename Type>
+struct PAC {
+  HDI Type operator()(Type in) { return ((1- myExp(-1*in))/(1+myExp(-1*in))); }

```

Comment:
I think it's numerically more stable to directly call `tanh(in / 2)`? @rietmann-nv What do you think?